test/pummel/test-fs-watch-non-recursive.js

```diff
@@ -38,13 +38,12 @@ const filepath = path.join(testsubdir, 'watch.txt');
 
 fs.mkdirSync(testsubdir, 0o700);
 
-// Need a grace period, else the mkdirSync() above fires off an event.
+const watcher = fs.watch(testDir, { persistent: true }, (event, filename) => {
```

Comment:
It might be good to add a comment explaining that the callback may be called one time or not at all, so it is a bug to wrap it in `common.mustCall()`. Otherwise, some well-meaning individual might come along later and try to wrap it in `common.mustCall()` or `common.mustCallAtLeast()`.

Reply:
Added comment below, good idea.